packer/ansible/roles/rackhd-ci-builds/tasks/main.yml

```diff
@@ -12,14 +12,52 @@
   sudo: yes
 
 - name: Install RackHD Packages latest
-  apt: pkg="{{ item }}" state=latest force=yes
+  shell: |
+    set timeout -1
+    spawn sudo aptitude install {{ item }}
+    expect {
+        "*Do you want to continue*" {send "y\r";set latest 1}
+        "*Accept this solution*" {send "n\r"; set latest 0}
+    }
+    if { $latest == 0 } {
+        expect "*Accept this solution*" {send "Y\r"}
+        expect "*Do you want to continue*" {send "Y\r"}
+       
+    }
+    expect { 
+        "*To continue*" {send "Yes\r"}
+    }
+    expect eof
+    catch wait result
+    exit [lindex $result 3]
+  args:
+    executable: /usr/bin/expect
   with_items:
     - rackhd
   sudo: yes
   when: (rackhd_version is undefined) or (not rackhd_version)
 
 - name: Install RackHD Packages specific version
-  apt: pkg="{{ item }}" force=yes
+  shell: |
```

Comment:
Thanks, there's an other way to do this:
* create a ```rackhd_install.exp``` and recieve a parameter ```version```
But I think 
**write shell in ansible yml dedirectly** 
is more clear than 
**scp the file to target vm and then run the shell**